### PR TITLE
Add epam email in profile

### PR DESCRIPTION
**Pull request by Pulya10c**
#### 🧑‍⚖️ Pull Request Naming Convention

 - Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
 - Do not put issue id in title
 - Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
 - Consider to add `area:*` label(s)

 * [x] I followed naming convention rules

---

#### 🤔 This is a ...
- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

https://github.com/rolling-scopes/rsschool-app/issues/940

#### 💡 Background and solution 

add epam EMAIL prop for profile

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

**Review by apalchys — APPROVED** [review]

File: client/src/components/Profile/ContactsCard.tsx [hunk 1]
```diff
@@ -100,6 +100,13 @@ class ContactsCard extends React.Component<Props> {
         key: 'linkedIn',
       },
     ];
+    if (epamEmail) {
+      contacts.splice(0, 0, {
+        name: 'EPAM E-mail',
+        value: epamEmail,
+        key: 'epamEmail',
+      });
```
Comment (apalchys) [inline client/src/components/Profile/ContactsCard.tsx:108]: Please consider to change it to `push`

```suggestion
      contacts.push({
        name: 'EPAM E-mail',
        value: epamEmail,
        key: 'epamEmail',
      });
```

Comment (Pulya10c) [reply client/src/components/Profile/ContactsCard.tsx:108]: We cannot use push, because then it will not be in the test where we expect it.

Comment (apalchys) [reply client/src/components/Profile/ContactsCard.tsx:108]: sorry, I don't get it. Could you please show where it will not be?

Comment (apalchys) [reply client/src/components/Profile/ContactsCard.tsx:108]: i just checked out the branch, changed it and seems it works

Comment (Pulya10c) [reply client/src/components/Profile/ContactsCard.tsx]: Thanks, fixed.

**Review by apalchys — APPROVED** [review]

**Pull request merged**